permits/services.py

```diff
@@ -1693,11 +1694,11 @@ def get_wot_properties(value, value_with_type=False):
             wot_properties = list()
             for prop in wot_props:
                 wot = f'{prop["works_object_type__works_object__name"]} ({prop["works_object_type__works_type__name"]})'
-                # List of a lost, to split wot in objects. Check if last wot changed or never assigned, means first iteration
+
+                # List of a list, to split wot in objects. Check if last wot changed or never assigned. Means it's first iteration
                 if property and wot != last_wot:
                     wot_properties.append(property)
                     property = []
-
                     # WOT
                     property.append(
```

Comment:
No need to check if user is authenticated here ?

Reply:
No, if we want to show the public `WOT` without any properties. It only shows public requests, cause the queryset selects only the ones the user can see